chat-android/src/main/java/com/ably/chat/Discontinuities.kt

```diff
@@ -42,6 +43,13 @@ public interface EmitsDiscontinuities {
     }
 }
 
+/**
+ * @return [ConnectionStatusChange] events as a [Flow]
+ */
+public fun EmitsDiscontinuities.discontinuityAsFlow(): Flow<ErrorInfo?> = transformCallbackAsFlow {
```

Comment:
I didn't work on flows yet. 
So, had a question if there are libraries returning `cold` flows. 
I am thinking, most kotlin SDKs return shared flows, so instance can be shared by multiple subscribers.

Reply:
Okay, I asked perplexity, and it seems most kotlin public APIs return `cold` flows by default : )
So, we are okay there, but you can double check 👍

Reply:
Btw, feature level discontinuity will be move to `room` level. So, we will not need this

Reply:
Maybe you like to remove `discontinuityAsFlow` for feature to avoid introducing breaking public API changes.

Reply:
We are not going to do next release before single channel is landed, so there'll be no breaking changes for users. I think we even should preserve interface, it's generic enough, maybe some other API will have discontinuities as well

Reply:
Okay, then we will update once single channel changes are introduced